File: src/app/patient/router/registerAppointment.kt
package app.patient.router

import io.ktor.routing.*
import io.ktor.application.*
import io.ktor.http.*
import io.ktor.response.*

fun Route.registerAppointment() {
    post("/api/patient/appointments") {
        call.respond(HttpStatusCode.OK)
    }
}

File: client/patient/src/api/registerAppointment.js
import { getUnixTime } from "date-fns";

export const registerAppointment = async ({ doctorId, date }) => {
    await fetch("/api/patient/appointments", {
        method: "POST",
        headers: {
            'Content-Type': 'application/json'
        },
        body: JSON.stringify({
            doctorId,
            date: getUnixTime(date) * 1000
        })
    })
}
